scripts/align-version.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
```

Comment:
Should we extract this from CDK into a stupid little node module instead of copying and pasting?

Reply:
I'm cool with it. Cute name ideas?